onnx/tools/net_drawer.py

```diff
@@ -61,18 +61,20 @@ def ReallyGetOpNode(op: NodeProto, op_id: int) -> pydot.Node:
             node.set_URL(url)
         return node
 
-    return ReallyGetOpNode
+    return really_get_op_node
 
 
-def GetPydotGraph(
+def get_pydot_graph(
```

Comment:
Not sure whether it is safe to change the public API name here. Possibly some users might rely on these APIs already?

Reply:
If you like I can revert this change ?

Reply:
Yes, I think it's safer. And please add a TODO comment to change it in next major version bump. Thank you!

Reply:
Done

Reply:
Actually, this is a commandline tool. I don't think anyone will use the functions themselves? If they do personally I think it's still a minor break?

Reply:
(also, tools/executables (anything with a `main()`) shouldn't be imported so implementation details are expected to change imo)

Reply:
I see ok! Thanks for doing the search. I will revert in this case. I don't think it's worth changing this just for the naming. Although we should not have exposed functions in a file with `__main__`